posts/2023-07-05-regex-1.9.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "Announcing regex 1.9"
+author: Andrew Gallant
+---
+
+The regex sub-team is annoncing the release of `regex 1.9`. The `regex` crate
```

Comment:
If this is on behalf of a subteam, maybe we should be linking it in the author line? You can look for examples in other posts for the formatting we parse.

(I guess it is just you right now, so not a huge difference in practice).

Reply:
Yeah, that sounds right.